internal/service/finspace/kx_cluster_test.go

```diff
@@ -58,6 +59,72 @@ func TestAccFinSpaceKxCluster_basic(t *testing.T) {
 	})
 }
 
+func TestAccFinSpaceKxCluster_invalidEnvironmentId(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping long-running test in short mode")
+	}
+
+	ctx := acctest.Context(t)
+	var kxEnvironment finspace.GetKxEnvironmentOutput
+	rName := sdkacctest.RandomWithPrefix(acctest.ResourcePrefix)
+	invalidEnvironmentId := sdkacctest.RandString(22)
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acctest.PreCheck(ctx, t)
+			acctest.PreCheckPartitionHasService(t, finspace.ServiceID)
+		},
+		ErrorCheck:               acctest.ErrorCheck(t, finspace.ServiceID),
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories,
+		CheckDestroy:             testAccCheckKxClusterDestroy(ctx),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccKxClusterConfigBase(rName),
```

Comment:
Do we need this first test step (creating an environment)? Seems like we could just create a vpc and try creating a cluster with an invalid env id

Reply:
The reasoning was to confirm that an environment was created before doing some negative testing. I did try it without this step and it works completely fine too! What should be our next steps?

Reply:
Ah gotcha, that's fair. I'm just not sure if it's necessary if we already have the basic environment creation test as well, but perhaps Mayank or Josh can give more insight on that